frontend/src/components/Table/Table.tsx

```diff
@@ -390,7 +414,7 @@ const Table = <T extends object>(props: PropsWithChildren<TableProps<T>>): React
       );
     return (
       props.detailsPanel && (
-        <TooltipWrapper toolTipId="expand-all-rows" toolTip={open ? 'Collapse Row' : 'Expand Row'}>
+        <TooltipWrapper toolTipId="filter-rows" toolTip={open ? 'Collapse Row' : 'Expand Row'}>
```

Comment:
the tooltip text should probably be update for this as well right?

Reply:
Ooops, this did not have to change